src/cipher.jl

```diff
@@ -89,7 +89,11 @@ end
 
 function trim_padding_PKCS5(data::Vector{UInt8})
   padlen = data[sizeof(data)]
```

Comment:
Why `data[sizeof(data)]` and not just `data[end]`? Since it is a `Vector{UInt8}`, it does not matter, does it?

Reply:
I didn't write this code, but I agree, it should be `data[end]`. It shouldn't have any real effect, but to better match the style of idiomatic Julia code, it should be changed.  We can do it in a different PR though.